docker-compose.prod.yml

```diff
@@ -9,12 +9,18 @@ services:
       - 8000
     env_file:
       - ./.env.prod
+    environment:
+      - DEBUG
   db:
     image: postgres:12.0-alpine
     volumes:
       - postgres_data:/var/lib/postgresql/data/
     env_file:
       - ./.env.prod.db
+    environment:
+      - POSTGRES_DB
```

Comment:
not needed if exist in `.env`

Reply:
Docker isn't automatically passing all env variables to all containers